.prettierrc

```diff
@@ -1,4 +1,5 @@
 {
   singleQuote: true,
-  printWidth: 80
+  printWidth: 80,
+  quoteProps: preserve
```

Comment:
I'm surprised to see a prettier change included in this merge.  I assume it's intentional?  Do we need to re-run prettier on the repo now?  (maybe not, since "preserve" apparently respects the existing code?)

Reply:
@hsubox76 Do you remember why we need to introduce `quoteProps: preserve`?